app/src/chainlit.py

```diff
@@ -225,3 +243,29 @@ def _get_retrieval_metadata(result: OnMessageResult) -> dict:
         ],
         "raw_response": result.response,
     }
+
+
+async def _batch_proccessing(file: AskFileResponse) -> None:
+    await cl.Message(
+        author="backend",
+        content="Received file, processing...",
+    ).send()
+
+    try:
+        engine: chat_engine.ChatEngineInterface = cl.user_session.get("chat_engine")
+        result_file_path = await batch_process(file.path, engine)
+
+        # E.g., "abcd.csv" to "abcd_results.csv"
+        result_file_name = file.name.rsplit(".csv", 1)[0] + "_results.csv"
```

Comment:
`.removesuffix()` is available
```suggestion
        result_file_name = file.name.removesuffix(".csv") + "_results.csv"
```

Reply:
TIL!